src/util-base64.c

```diff
@@ -103,7 +103,6 @@ uint32_t DecodeBase64(uint8_t *dest, const uint8_t *src, uint32_t len) {
             /* Invalid character found, so decoding fails */
             if (src[i] != '=') {
                 valid = 0;
-                numDecoded = 0;
```

Comment:
What effects (if any) does this have on other users of DecodeBase64? Seems SMTP decoder uses it.